MC/config/common/external/generator/TPCLoopers.C

```diff
@@ -3,6 +3,8 @@
 #include <vector>
 #include <fstream>
 #include <rapidjson/document.h>
+#include "CCDB/BasicCCDBManager.h"
```

Comment:
I don't see BasicCCDBManager used anywhere.

Reply:
Indeed, I used it before for other tests. Now replaced with CCDBTimeStampUtils.h